account_payment_order/tests/test_payment_order_inbound.py

```diff
@@ -77,6 +77,7 @@ def _create_customer_invoice(self):
                 invoice_line_form.price_unit = 100.0
                 invoice_line_form.account_id = self.invoice_line_account
                 invoice_line_form.tax_ids.clear()
+        invoice_form.reference_type = "structured"
```

Comment:
The fact that you need to change this means that there's a breaking change. Can you please avoid it?

Reply:
I made this change in a separate commit to demonstrate how to reproduce the bug, but it is not necessary.

Reply:
OK, understood. Then just squash both together and indicate in the extended message commit that the test is modified to be able to reproduce the problem.

Reply:
done